easybuild/easyconfigs/q/Qt5/Qt5-5.15.10_fix-qtwebengine-breakpad-glibc-2.34.patch

```diff
@@ -0,0 +1,11 @@
+--- a/qtwebengine/src/3rdparty/chromium/third_party/breakpad/breakpad/src/client/linux/handler/exception_handler.cc	2023-08-14 14:19:20.751150000 +0200
```

Comment:
Please add a patch description. Preferably include what error you get without the patch, how this fixes it, and referencing upstream issues if there are any. That should help us determine for future versions if the patch is still needed (e.g. by checking if the upstream issue is still open, or is merged with references to a fix).

Reply:
I have submitted the description to my branch. Do I need to do something to retrigger the test pipeline with?

Reply:
Nope, CI retriggers automatically after a push to the feature branch in this PR. Thanks!